pallets/blogchain/src/lib copy.rs

```diff
@@ -0,0 +1,102 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
what's the point of this file? :thinking:

Reply:
Oh, just realised I included also all my temp files. Sorry, nice note!